api/openapi.yaml

```diff
@@ -412,6 +412,6 @@ paths:
         200:
           description: OK
           content:
-            application/yaml:
+            application/json:
               schema:
-                type: string
\ No newline at end of file
+                $ref: "./tests.yaml#/components/schemas/DefinitionFile"
```

Comment:
Well, openAPI doesn't allow us to define a `yaml` endpoint, so I'm wrapping it in a JSON object

Reply:
What happens when you use yaml? Isn't there a workaround this? If not, I'd put this outside openapi, or change the endpoint, because having a `/definition.yaml` endpoint that actually returns a json containing a yaml string is super confusing

Reply:
Yeah, it doesn't make sense. The only workaround would be writing the handler ourselves. The problem is that our HTTP clients would not contain that method as well :(